examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -65,8 +65,9 @@
    },
```

Comment:
<p>tried running the notebook first without making any major changes (except using numpy random Generator for setting the seed), but I get an error here. The <code>summary_multinomial</code>  dataframe that I get is not the same as before, because,</p><p>creation of the column <code>ess_mean_per_sec</code> depends on a column called <code>ess_mean</code>. However, here we don't get the columns <code>ess_mean</code>  and <code>ess_sd</code> , is there something obvious I missed here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>arviz summary now uses <code>ess_bulk</code>  as default. They are not exactly the same but nearly. For the purposes of these notebook we can just change <code>ess_mean</code>  for <code>ess_bulk</code>  </p>